### ISSUE TITLE
iOS 16 Xcode14 Thread Performance Checker: Thread running at QOS_CLASS_USER_INTERACTIVE waiting on a lower QoS thread running at QOS_CLASS_DEFAULT.

### ISSUE THREAD

**Issue by TmacCL0523** [facebookincubator/SocketRocket#648]
This issue appears in version 0.60. IOS version 16.0 Xcode version 14.0

<img width="1835" alt="企业微信截图_2e745736-65f6-4161-8f4e-fbb4fe4b4168" src="https://user-images.githubusercontent.com/64182079/190042763-09f6e080-e0db-4fdc-bb8f-7a6746aff65d.png">

**Comment by wangkai598** [comment]
I also met

**Comment by wangjie172767** [comment]
How to solve it?

**Comment by bolo992** [comment]
When this PR will be merged?

**Comment by samsiner** [comment]
> When this PR will be merged?

@nlutsenko I believe you have write access. Could you please review the PR listed above and merge if approved?

**Comment by KeiroMidori** [comment]
Any update on this?

**Comment by samsiner** [comment]
No, still waiting for @nlutsenko or someone with write access to merge the PR.

**Comment by zonggexu** [comment]
**Hello, how to solve this problem, is there a new update?** @samsiner

**Comment by wzJun1** [comment]
perfect!

**Comment by rgmyyw** [comment]
+1

**Comment by mmatiush** [comment]
Any available solution?

**Comment by mmatiush** [comment]
Updating SocketRocket to 0.7.0 version fixed the issue
